atk/Cargo.toml

```diff
@@ -31,7 +31,7 @@ features = ["dox"]
 libc = "0.2"
 bitflags = "1.0"
 ffi = { package = "atk-sys", path = "sys" }
-glib = { git = "https://github.com/gtk-rs/gtk-rs-core" }
+glib = "0.17"
```

Comment:
You want to keep git

```suggestion
glib = { git =  "https://github.com/gtk-rs/gtk-rs-core", branch = "0.17", version = "0.17" }
```

etc

Reply:
Oh? Ok.

Reply:
@bilelmoussaoui also missed that in gtk4

Reply:
Fixed. :)